multiversx_sdk_cli/downloader.py

```diff
@@ -9,9 +8,9 @@
 logger = logging.getLogger("downloader")
 
 CHUNK_SIZE = 1024 * 16
-LINECLEAR = '\r' + ' ' * 20 + '\r'
+LINECLEAR = "\r" + " " * 20 + "\r"
```

Comment:
Maybe, in a future PR, replace this logic with the package `wget`?

Reply:
will keep it in mind to replace it in a future PR.